specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json

```diff
@@ -2660,6 +3369,26 @@
       ],
       "x-ms-discriminator-value": "KeyPhraseExtractionLROResults"
     },
+    "AnalyzeConversationLROResult": {
```

Comment:
The "LRO" acronym likely won't pass the SDK review board (we try to avoid non-common acronyms). Could this maybe be something like "AnalyzeConversationOperationResult"?

Reply:
Updating to AnalyzeConversationResult.